src/theory/quantifiers/fmf/bounded_integers.cpp

```diff
@@ -451,11 +452,19 @@ void BoundedIntegers::checkOwnership(Node f)
           success = true;
           //set Attributes on literals
           for( unsigned b=0; b<2; b++ ){
-            if( bound_lit_map[b].find( v )!=bound_lit_map[b].end() ){
-              Assert( bound_lit_pol_map[b].find( v )!=bound_lit_pol_map[b].end() );
+            if (bound_lit_map[b].find(v) != bound_lit_map[b].end())
+            {
+              // This assertion may fail. We intentionally do not enable this in
+              // production as it is considered safe for this to fail. We fail
+              // in debug mode to have this instances raised to our attention.
+              Assert(bound_lit_pol_map[b].find(v)
+                     != bound_lit_pol_map[b].end());
```

Comment:
This would be a candidate for the future "warning" (or whatever we will name it) macro, right? (Referring to our discussion on Wednesday, here.) Any ideas how we could keep track of newly introduced ones that are still Asserts since we don't have the macro in place yet?

Reply:
I added WARNING_CANDIDATE as a comment. This should make tracking easy via grep.